File: Example/PrebidDemoJava/src/main/java/org/prebid/mobile/javademo/testcases/AdFormat.java
package org.prebid.mobile.javademo.testcases;

public enum AdFormat {

    DISPLAY_BANNER("Display Banner"),
    VIDEO_BANNER("Video Banner"),
    DISPLAY_INTERSTITIAL("Display Interstitial"),
    VIDEO_INTERSTITIAL("Video Interstitial"),
    VIDEO_REWARDED("Video Rewarded"),
    IN_STREAM_VIDEO("In-stream Video"),
    NATIVE("Native");

    private final String description;

    AdFormat(String description) {
        this.description = description;
    }

    public String getDescription() {
        return description;
    }

}

File: Example/PrebidDemoKotlin/src/main/java/org/prebid/mobile/prebidkotlindemo/testcases/AdFormat.kt
package org.prebid.mobile.prebidkotlindemo.testcases

enum class AdFormat(
    val description: String
) {

    BANNER_DISPLAY("Display Banner"),
    BANNER_VIDEO("Video Banner"),
    INTERSTITIAL_DISPLAY("Display Interstitial"),
    INTERSTITIAL_VIDEO("Video Interstitial"),
    REWARDED("Rewarded Video"),
    IN_STREAM_VIDEO("In-stream Video"),
    NATIVE("Native"),

}